src/Extensions.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Wasmtime
+{
+    internal static class Extensions
+    {
+        public const UnmanagedType LPUTF8Str =
+#if NETSTANDARD2_0
+            (UnmanagedType)48;
```

Comment:
Yeah, I was wondering about that too. I think .NET Standard only governs the presence of a particular API, but not necessarily its full implementation, i.e. there are situations where a "supported" API actually throws `PlatformNotSupportedException`, so in my opinion it is fine to ignore this for the moment. The complete solution would probably be to copy those 6 methods that use it with a custom marshaller and use the original ones only if `(UnmanagedType)48` is actually defined at runtime. A bit complicated, but it is up to you if you feel it would be better.

Reply:
Thanks! I'm OK with ignoring it for the moment.